tests/e2e/utils/process.ts

```diff
@@ -320,3 +320,11 @@ export function deprecate(pkg: string, msg: string): Promise<any> {
       .catch(() => reject());
   });
 }
+
+export function npmUnPublish(pkg: string): Promise<any> {
+  return new Promise((resolve, reject) => {
+    exec('npm', ['-q', 'unpublish', pkg, '--force'])
```

Comment:
Unpublish has to be called with `--force`. Otherwise, npm says `npm ERR! Refusing to delete entire project. npm ERR! Run with --force to do this.`